src/canvas.class.js

```diff
@@ -1243,6 +1243,11 @@
         if (obj.onDeselect({ e: e, object: object })) {
           return false;
         }
+        if (this._currentTransform && this._currentTransform.target === obj) {
+          this._finalizeCurrentTransform(e);
+          this._currentTransform = null;
```

Comment:
should we call `this._resetTransformEventData();`?

Reply:
@asturur this is not outdated
Thoughts?

Reply:
It's really `Canvas._target` , I think it should be reset but I'm not sure

Reply:
I have encountered the case that demands to nullify `_target`
When an object is being moved by the user and discarded meanwhile.
Currently the object will continue moving and will be inactive only on mouseup.

Reply:
Note for future self

setting `canvas._target` on `mousedown:before` hijacks the event and fires it on the hijacker

Reply:
#8335